lib/tasks/repopulate_pdf_count.rake

```diff
@@ -1,15 +0,0 @@
-namespace :temp do
-  desc "Update withdrawn editions"
-  task repopulate_pdf_count: :environment do
-    Dimensions::Edition.latest.relevant_content.includes(:publishing_api_event).in_batches do |group|
-      group.each do |edition|
-        if edition.publishing_api_event
-          pdf_count = Etl::Edition::Metadata::NumberOfPdfs.parse(edition.publishing_api_event.payload)
-          edition.facts_edition.update!(pdf_count: pdf_count)
-        end
-      end
-      putc '.'
-    end
-    puts '\nDone!\n'
-  end
-end
```

Comment:
Why is this being removed?

Reply:
This was a one-off `rake task` to repopulate the pdf count. It does not have a test so I am not feeling comfortable with having rake tasks with no associates tests as we can't guarantee that they are going to work in the mid - long run. 

I would rather removing it, and it we think we need a rake task to repopulate PDF, or other attributes, they we can create one with that purpose in mind (and document and test accordingly)

Reply:
Just making sure I understand - you would have need modify this rake task to make it compatible with the rest of changes in this PR - but because it doesn't have any tests, you'd rather remove it than make the changes and add tests?

Reply:
The main reason is not having rake tasks that were created for a migration (a one-off thing) and left without a clear purpose nor proper tests.

I think we should remove any rake task that has no tests, nor is intended for a clear use case.

Reply:
Okay, in which case it seems out of scope for this PR